packages/super-components/components/msc-billing-tile/src/msc-billing-tile/Commitment.tsx

```diff
@@ -0,0 +1,131 @@
+import { h } from '@stencil/core';
+import {
+  OdsThemeColorIntent,
+  OdsThemeTypographyLevel,
+  OdsThemeTypographySize,
+} from '@ovhcloud/ods-theming';
+import {
+  OdsIconName,
+  OdsIconSize,
+  OdsChipSize,
+  OdsHTMLAnchorElementTarget,
+  OdsChipVariant,
+} from '@ovhcloud/ods-core';
+import formatDate from '../../../../utils/date.utils';
+import { getTranslation } from '../../../../utils/translation.utils';
+
+export const CommitmentContent = (
```

Comment:
why not creating with a class ? (export class) like the other ?

and the name of the file msc-billing-commitment ?